### DISCUSSION TITLE
Doku bzgl. Installation K8s / Docker erweitern

### DISCUSSION THREAD

**Discussion by hildebrandttk** [OpenCCU/OpenCCU#1175] [🇩🇪 Ideen]
Hallo,

ich habe eine Test-Installtion abschließen können, allerdings erst nachdem ich aus dem deploy.sh Skript aus der Docker-Anleitung die Infos zu den udev-Rules rausgesucht habe. Vorher ist mein Funkstick nicht gefunden worden. Dabei ist mir aufgefallen, dass die auch die Docker-Doku bis auf dem Aspekt alle Teile aus deploy.sh beschreibt.

**Comment by jens-maus** [comment]
Und wo ist jetzt genau die Frage? ;)

**Reply by hildebrandttk** [reply]
Eher ne Feststellung und die Frage ob man die Doku in die Richtung erweiten kann, damit es eine vollständige manuelle Installation gibt und alternativ das deploy.sh-Skript.

**Reply by jens-maus** [reply]
Ok, dann schieb ich das mal in die "Ideen" Rubrik hier. Und es sei nebenbei noch einmal darauf hingewiesen, dass die Dokumentation ein frei zugänglicher Wiki ist. D.h. jeder kann die Dokumentation editieren (nach GitHub login) und entsprechend anpassen und verbessern! Wenn also etwas fehlt oder unklar ist, einfach anpassen...
